src/libs/ReportUtils.ts

```diff
@@ -346,6 +347,9 @@ type OnyxDataTaskAssigneeChat = {
     optimisticChatCreatedReportAction?: OptimisticCreatedReportAction;
 };
 
+let environmentURL: string;
+Environment.getEnvironmentURL().then((url: string) => (environmentURL = url));
+
```

Comment:
Can you see tryResolveUrlFromApiRoot instead of this?

Reply:
Maybe above method doesn't work in that case, thats why they have used environmentURL directly.